pressbooks/pressbooks
# fix: restores scroll behavior to my books dropdown

Issue #3231 

This PR restores the scroll behaviour to the "My Books" dropdown.

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/1761690/224090337-525d9e45-01ec-41ed-8cf5-01a7929260eb.png">

**How to test**

1. Add the same user to multiple books
2. Hover the "My Books" dropdown and make sure the list is scrollable if not all books are showing

> **Note**
> If you don't have a good amount of books in your local network you can use the following snipped to mimic a large list of books. Simply replace [these lines](https://github.com/pressbooks/pressbooks/blob/3f8f9dfdb4266e94eeeae635fe6fe4e8243aff80/inc/admin/menus/class-topbar.php#L221-L226) in the `class-topbar.php` with the following code:
```php
foreach (range(1, 50) as $id) {
	$bar->add_node([
		'parent' => 'pb-my-books-list',
		'id' => "book-{$book->userblog_id}-{$id}",
		'title' => "<span class='blavatar'></span> {$title}",
		'href' => get_admin_url($book->userblog_id),
	]);
}
```
